plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/views/constraintview/ConstraintViewController.java

```diff
@@ -250,6 +254,7 @@ private void addListeners() {
 	 * a FeatureDiagram" text is shown.
 	 */
 	public void refresh() {
+		System.err.println("REFRESH CALLED");
```

Comment:
Either remove or change to Logger.logInfo(...)

Reply:
Oh :-D good catch